tests/unit/test_configuration.py

```diff
@@ -0,0 +1,24 @@
+import ConfigParser
+import StringIO
+
+import pytest
+
+from twindb_backup.configuration import get_destination
+from twindb_backup.destination.ssh import Ssh
+
+
+def test__get_destination_ssh_valid_port(config_content):
+    s_config = config_content.format(destination="ssh", port=22)
```

Comment:
can you add one more test?

`s_config = config_content.format(destination="ssh", port='1234')`